mission-vision-values.md

```diff
@@ -132,7 +132,7 @@ acknowledging that people come from different backgrounds and cultures.
 There might be situations where community members act in a dubious manner. If
 you have seen or experienced unacceptable behavior or anything that would make
 our community less welcoming, please speak up! See
-[our code of conduct](https://github.com/open-telemetry/community/blob/main/code-of-conduct.md)
+[our code of conduct](./code-of-conduct.md)
```

Comment:
This update causes the following issue (FYI), because the (now relative) link is outside of the OTel website:

- https://github.com/open-telemetry/opentelemetry.io/issues/6561

Reply:
@chalin is this something that would be good to fix on the community repo side?

Reply:
@trask: yes - https://github.com/open-telemetry/opentelemetry.io/issues/6561#issuecomment-2824843553.